src/app/modules/timesheet/components/timesheet-edit/timesheet-edit.component.ts

```diff
@@ -167,5 +164,6 @@ export class TimesheetEditComponent implements OnInit {
     this.generateInvoice = !!timesheet.invoice && !_.isEqual(timesheet.invoice, Object.assign({}, new Invoice()));
     this.generateExpenses = timesheet.commutes.length > 0 || timesheet.flatFees.length > 0 || timesheet.miscellaneous.length > 0;
     this.setShortUrl();
+    this.onUserInput();
```

Comment:
It's strange to say there is an user input when you load the timesheet.

Then you can remove this function call and replace at the first method line `this.showLinks = true;` by `this.showLinks = false;`